src/lua/compat.lua

```diff
@@ -4,19 +4,40 @@
 local NEW = true
 local OLD = false
 
-local internals = require('compat_internals') -- luacheck: no unused
+local internals = require('compat_internals')
 
 local options_format = {
     default  = 'string',
     brief    = 'string',
     obsolete = 'boolean',
 }
 
+local JSON_ESCAPE_BRIEF = [[
+Whether to escape the forward slash symbol '/' using a backslash in a json.encode()
+result. The old and the new behavior produce a result, which is compatible
+with the JSON specification. However most of other JSON encoders don't escape
+the forward slash, so the new behavior is considered more safe.
+https://github.com/tarantool/tarantool/wiki/compat%3Ajson_escape_forward_slash
+]]
+
 -- Contains static options descriptions in format specified by `options_format`.
-local options = { }
+local options = {
+    json_escape_forward_slash = {
+        default = 'old',
+        obsolete = false,
+        brief = JSON_ESCAPE_BRIEF,
+    },
+}
 
 -- Contains postaction functions for each non obsolete option from `options`.
-local postaction = { }
+local postaction = {
+    json_escape_forward_slash = function(value)
+        value = not value
```

Comment:
Isn't the value either `'old'` or `'new'`?

Reply:
No. Action should handle only boolean values, `true` for `'new'`; `false` for `'old'`.

Reply:
We identify `'new'` as `true` in other parts of compat too (see `NEW` and `OLD` vars). We would have to convert 'new' and 'old' to bool in every postaction anyway, thus I don't see the point.

Reply:
I've consulted with @igormunkin on the subject, and he agrees.

Reply:
No problem, okay. It would be good to have it documented somewhere for module developers.